articles/api/management/v2/tokens.md

```diff
@@ -1,86 +1,278 @@
+  :::
+
+Go to the [Clients section](${manage_url}/#/clients) of the dashboard and click the **Create Client** button.
+
+Enter a name for your new client, select **Non Interactive Clients** and then click **Create**.
 
 ![Create New Client](/media/articles/api/tokens/noninteractive-client.png)
 
-**Authorize the Client**
+### 2. Authorize the Client
+
+Once you create the new client, you will be navigated to it's *Quick Start* section.
+
+If this is the first time you are configuring an API in Auth0, you will see this message:
+
+```text
+Defining Non Interactive Clients and APIs is a new feature in Auth0 that you can opt-in to.
+Once you enable it, a new item on the sidebar "APIs" will be shown.
+To enable it, turn on this toggle (or do it on Account Settings).
+```
+
+If you see that, turn on the toggle. Notice that a new item is added on the left hand menu: [APIs](${manage_url}/#/apis).
+
+Now you should see a `Select an API` dropdown, listing all the APIs you have configured with Auth0.
 
-After creating the new client, you will be brought to it's **Quick Start** section. Select **Auth0 Management API** from the dropdown. You will then see a message that the client is not authorized, click **NAVIGATE TO THE API AND AUTHORIZE**.
+Select `Auth0 Management API` from the dropdown. You will then see a message that the client is not authorized.
 
-![](/media/articles/api/tokens/navigate-button.png)
+Click **Navigate to the API and authorize**.
 
-This will bring you to the [APIs section](${manage_url}/#/apis) under **Auth0 Management API** in the **Non Interactive Clients** section. (To enable the APIs section of the sidebar, go to Account Settings > Advanced > Enable APIs Section)
+![Navigate to the API and authorize button](/media/articles/api/tokens/navigate-button.png)
 
-![Authorize Client](/media/articles/api/tokens/authorize-noninteractive.png)
+This will bring you to the [APIs section](${manage_url}/#/apis), specifically to the _Non Interactive Clients_ tab of the `Auth0 Management API`.
 
-Toggle the slider to authorize your client.
+![Authorize Non Interactive Client](/media/articles/api/tokens/authorize-noninteractive.png)
 
-**Choose the scopes**
+Toggle the slider from `Unauthorized` to `Authorized` for your client.
 
-You will see the available scopes that you can be granted for this client (which can be revoked at any time). Choose the desired scopes and then click the **UPDATE** button.
+### 3. Choose the Scopes
+
+The last step, before you get a token, is to select which scopes should be granted to this client.
```

Comment:
We should tell the customers that if they need different sets of scopes for different applications they plan to call the mgmt api from, they better create a new non-interactive client for each.

For example:
NIC1: create:clients, read:clients
NIC2: create:users,read:users

Instead of having just one non interactive client with all the previous scopes and re-use it in different clients (applications).